index.html

```diff
@@ -265,6 +207,66 @@ <h2 >Consult A Doctor</h2>
       </div>
   </section>
 
+  <!-- About Section -->
+  <section id="about">
+    
+        <h2>About Us</h2>
+        <h3><span style="color: rgb(91, 134, 150);">MEET OUR MEMBERS</span></h3>
+      
+
+      <div class="box">
+
+        <div class="card">
+          <img src="/images/shayan.jpeg" alt="shayan" loading="lazy">
+          <h5>Shayan Pal</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://bio.link/shayanpal"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/soumyadeep.jpeg" alt="soumyadeep"loading="lazy">
+          <h5>Soumyadeep Dhali</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://soumyadeep.bio.link"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/susmita.jpg" alt=""loading="lazy">
+          <h5>Susmita Dey</h5>
+          <div class="pra">
+            <p>Head of The Team</p>
+            <a href="https://bio.link/susmitadey"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/chaitri.jpeg" alt="chaitri"loading="lazy">
+          <h5>Chaitri Saha Chowdhury</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://bio.link/chaitri"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/sasanka.jpeg" alt="sasanka"loading="lazy">
+          <h5>Sasanka Kundu</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://bio.link/sasankakundu"><button>Let's Connect</button></a>

```

Comment:
If you are using a tag, There is no need use button inside of it.
You can design it to look like a button.
```diff
- <a href=""><button></button></a>
+ <a href=""></a>
```
> **Note**
> For styling a tag, If you want to add `padding` on it, set the display to equal to inline-block.

> **Warning**
> If you don't use inline-block, The `padding-top` and `padding-bottom` is not going to work.